src/pages/reports/comparativeReport/AddEmployee.tsx

```diff
@@ -0,0 +1,121 @@
+import React, {useState} from 'react'
+import {Box, Button, FormControl, IconButton, Paper, TextField, Typography} from '@mui/material'
+import useCreateEventStyles from '../../../assets/styleJs/events/createEvent'
+import {v4 as uuidv4} from 'uuid'
+import DeleteIcon from '../../../assets/images/Icons/DeleteIcon'
+import useReportsStyle from '../../../assets/styleJs/report/report'
+import {randomColor} from '../../../utils/utils'
+import {IEmployee} from '../../../types/eployeeTypes'
+
+const AddEmployee = () => {
+  const classesCreateEvent = useCreateEventStyles()
+  const classes = useReportsStyle()
+  const [employee, setEmployee] = useState<IEmployee[]>([
+    {
+      id: uuidv4(),
+      name: '',
+      position: '',
+      employee: 'Employee 1',
+      bgColor: randomColor(),
+    },
+    {
+      id: uuidv4(),
+      name: '',
+      position: '',
+      employee: 'Employee 2',
+      bgColor: randomColor(),
+    },
+  ])
+
+  const handleAddEmployee = () => {
+    const tempArr = []
+    const employeeNumber = employee.length + 1
+    tempArr[0] = {
+      id: uuidv4(),
+      name: '',
+      position: '',
+      employee: 'Employee ' + employeeNumber,
+      bgColor: randomColor(),
+    }
+    setEmployee([...employee, ...tempArr])
```

Comment:
Don't clear approach, What is the purpose of having tempArr? I don't see any usage in other places